prov/efa/src/rxr/rxr_pkt_type.h

```diff
@@ -231,6 +231,26 @@ void rxr_pkt_handle_receipt_send_completion(struct rxr_ep *ep,
 void rxr_pkt_handle_receipt_recv(struct rxr_ep *ep,
 				 struct rxr_pkt_entry *pkt_entry);
 
+/* General packet type helper functions */
+static inline
+int rxr_pkt_type_contains_rma_iov(int pkt_type)
+{
+	if (pkt_type == RXR_EAGER_RTW_PKT ||
```

Comment:
I think you can do `return (pkt_type == xxx || pkt_type == xxx || ...)`

Reply:
probably cleaner to just use a switch statement here